lib/ReactGridLayout.jsx

```diff
@@ -429,31 +461,17 @@ export default class ReactGridLayout extends React.Component<Props, State> {
           if (Number.isFinite(leastY)) l.h = leastY - l.y;
         }
       }
+      l.w = w;
```

Comment:
Doesn't this make L452-L461 essentially dead code?

Reply:
Yes, thanks for catching. 452-461 _shouldn't_ be needed anymore after testing, looks like the boundaries are still being calculated correctly.